src/formatting_stack/reporters/impl.clj

```diff
@@ -17,3 +17,19 @@
                                        (string/replace (re-pattern (str "^" (System/getProperty "user.dir")))
                                                        "")
                                        (string/replace #"^/" ""))))
+
+(speced/defn truncate-line-wise [^string? s, length]
+  (if (= s "\n")
```

Comment:
Whats with this special case?

Reply:
It's here to make sure` "\n"` isn't morphed into `""`:
```clj
user=> (clojure.string/join "\n" (clojure.string/split "\n" #"\n"))
""
```

Not sure what the origin is (or when `"/n"` would be passed into this fn to begin with). Would need to check.